### ISSUE TITLE
Can't install in the latest flutter version

### ISSUE THREAD

**Issue by gernest** [google/uri.dart#29]
I am trying to install this library.

My flutter version is 

```
Flutter 0.1.4 • channel beta • https://github.com/flutter/flutter.git
Framework • revision f914e701c5 (3 weeks ago) • 2018-02-19 21:12:17 +0000
Engine • revision 13cf22c284
Tools • Dart 2.0.0-dev.27.0-flutter-0d5cf900b0
```

I keep getting this message 

```
Package uri has no versions that match >=0.11.1 <0.12.0 derived from
```

**Comment by YASSERRMD** [comment]
Same here, i am trying add map_view package which depends on URI.

**Comment by YASSERRMD** [comment]
Hey Guys

Do you find any fix

**Comment by isoos** [comment]
@kevmoo, @mkustermann: who is the current owner who can publish a new version with updated dependencies?

**Comment by kevmoo** [comment]
Seems I have powers here...

**Comment by kevmoo** [comment]
Given how little this is touched, I'm wondering if it should be mothballed and folks should copy out the code they need...

This package is very much not maintained...

**Comment by initiuminc1** [comment]
Same for me. The map_view package is dependent on this

**Comment by kevmoo** [comment]
@isoos – care to look at https://github.com/google/uri.dart/pull/30

**Comment by kevmoo** [comment]
Now published as 0.11.2
